azext_iot/digitaltwins/providers/resource.py

```diff
@@ -64,13 +65,33 @@ def create(
                 identity={"type": "SystemAssigned" if assign_identity else "None"},
                 public_network_access=public_network_access,
             )
+
             create_or_update = self.mgmt_sdk.digital_twins.create_or_update(
                 resource_name=name,
                 resource_group_name=resource_group_name,
                 digital_twins_create=digital_twins_create,
                 long_running_operation_timeout=timeout,
             )
 
+            def check_state(lro):
+                from time import sleep
+                instance = lro.resource().as_dict()
+                state = instance.get('provisioning_state', None)
+                retries = 0
+                while state == "Provisioning" and retries < MAX_ADT_CREATE_RETRIES:
+                    retries += 1
+                    sleep(1)
+                    lro.update_status()
+                    instance = lro.resource().as_dict()
+                    state = instance.get('provisioning_state', None)
+                if retries == MAX_ADT_CREATE_RETRIES:
+                    raise CLIError(
+                        "The resource has been created and has not finished provisioning. Please monitor the status of "
+                        "the Digital Twin instance using az dt show -n {} -g {}".format(
```

Comment:
Just a minor suggestion to help make the command stand out from the error
```suggestion
                        "the Digital Twins instance using: 'az dt show -n {} -g {}'".format(
```